Makefile

```diff
@@ -1335,8 +1340,9 @@ clean: ## Remove generated binaries, GitBook files, Helm charts, and Tilt build
 	$(MAKE) clean-tilt
 
 .PHONY: clean-kind
-clean-kind: ## Cleans up the kind cluster with the name $CAPI_KIND_CLUSTER_NAME
+clean-kind: ## Cleans up the kind cluster with the name $CAPI_KIND_CLUSTER_NAME and its local registry named $CAPI_KIND_REGISTRY_NAME
 	kind delete cluster --name="$(CAPI_KIND_CLUSTER_NAME)" || true
+	docker rm -f "$(CAPI_KIND_REGISTRY_NAME)"
```

Comment:
I'd not like to add this. For local development it is great to preserve the registry, otherwise we always have to re-pull.

I tend to use often `make clean-kind tilt-up` for example.

Reply:
I see. I added this because I thought we don't want to leave a dangling container after cleanup. I can drop this line if you think it's better to keep the registry running.